app/views/appointments/doctor_index.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>My Appointments</h1>
+  <div id="patient-appointments" class"container table-responsive">
+    <table id="find-doctors" class="table">
+      <thead>
+        <tr>
+          <th>DATE</th>
+          <th>TIME</th>
+          <th>NAME</th>
+          <th>STATUS</th>
+          <th>CONTROLS</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @appointments.each do |appointment| %>
+          <tr>
+            <td><%= appointment.datetime %></td>
+            <td><%= appointment.datetime %></td>
+            <td><%= appointment.user.name %></td>
+            <td><%= appointment.status %></td>
+            <% if appointment.status == "Pending" %>
+            <td><%= button_to "Confirm", doctors_appointment_path(appointment.id) + "?appointment[status]=Confirmed", class: "btn btn-primary form-control", method: "put" %></td>
+            <td><%= button_to "Reject", doctors_appointment_path(appointment.id) + "?appointment[status]=Rejected", class: "btn btn-danger form-control", method: "put" %></td>
```

Comment:
Here you are updating directly the appointment by sending the information in the params.

A cleaner way to do this it would be to create a separate method in the appointments_controller that allows to reject/confirm an appointment:

```ruby
  resources :users, only: [:index, :create] do
    resources :appointments, only: [:new, :create] do
      member do
        put 'confirm'
        put 'reject'
      end
    end
  end
```

```bash
confirm_user_appointment PUT    /users/:user_id/appointments/:id/confirm(.:format) appointments#confirm
 reject_user_appointment PUT    /users/:user_id/appointments/:id/reject(.:format)  appointments#reject
```

```erb
<%= button_to "Confirm", confirm_user_appointment(appointment.id)", class: "btn btn-primary form-control", method: "put" %>

<%= button_to "Reject", reject_user_appointment_path(appointment.id), class: "btn btn-danger form-control", method: "put" %>
```